### ISSUE TITLE
Console error (json format error)

### ISSUE THREAD

**Issue by kezhongfa** [acao/codemirror-json-schema#79]
Phenomenon: official website demo must appear.
![image](https://github.com/acao/codemirror-json-schema/assets/6939549/8045fc8c-b95b-4801-be50-db0f6124ee64)
Reproduction step: add "," to any value.
Desired: error message syntax highlighting.
![image](https://github.com/acao/codemirror-json-schema/assets/6939549/42d1bf88-2bf8-4956-90bc-6063aa81467b)

**Comment by acao** [comment]
are you using the "simple" or "custom" approach?

**Comment by kezhongfa** [comment]
> are you using the "simple" or "custom" approach?

simple. I'm just giving an example. The console shouldn't report errors, it has to show the user the error message and know how to change it.

**Comment by nowaylifer** [comment]
Encountered it too. Filtering out diagnostics with `from` and `to` properties that equals `undefined` seems to solve the problem.
```javascript
import { jsonParseLinter } from '@codemirror/lang-json';
import { jsonSchemaLinter } from 'codemirror-json-schema';

const lintJson = jsonParseLinter();
const lintJsonSchema = jsonSchemaLinter();

export function customJsonLinter() {
  return (view: EditorView) => {
    const parseErrors = lintJson(view);
    const schemaErrors = lintJsonSchema(view).filter(
      (diagnostic) => diagnostic.from !== undefined && diagnostic.to !== undefined
    );

    return [...schemaErrors, ...parseErrors];
  };
}
```

**Comment by acao** [comment]
good catch, this is a recently introduced regression (from fixing required property validation) that we don't test for yet!

**Comment by acao** [comment]
after closer debugging, it's related to the introduction of new error types in our dependency which handles validation. I have a branch that should fix this!
